install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1272,3 +1285,147 @@ common issues:
 [Note] Completed with no errors or warnings seen.
 ----
 ====
+
+[[manual-upgrades-advanced-topics]]
+== Advanced topics
+
+Blue-green deployments are a proven approach to reducing downtime caused while
+updating an environment.  This is done by creating a parallel environment on
+which the new deployment can be installed.  After the new deployment is
+verified traffic can be switched over with the option to rollback should a
```

Comment:
Consider rewording:
If a problem is detected and after the new deployment is verified, traffic can be switched over with the option to rollback.